docs/api.md

```diff
@@ -86,9 +86,10 @@ Type: [Object][29]
 * `disableMediaPreload` **[number][32]** Whether to disable media pre loading while ad is playing. If set to `true`, the player will start loading the content media just after the ad break will end (incase of preroll ad). This will also overrides `config.adsRenderingSettings.restoreCustomPlaybackStateOnAdBreakComplete` no matters its value and sets it to `false`. (optional, default `false`)
 * `vpaidMode` **[google.ima.ImaSdkSettings.VpaidMode][36]** Sets VPAID playback mode See usage also in [Handling VPAID Modes][37]. (optional, default `'ENABLED'`)
 * `adsRenderingSettings` **[google.ima.AdsRenderingSettings][38]** Defines parameters that control the rendering of ads. (optional, default `{ restoreCustomPlaybackStateOnAdBreakComplete:true, enablePreloading:false, useStyledLinearAds:false, useStyledNonLinearAds:true bitrate:-1, autoAlign:true }`)
-  * `adsRenderingSettings.useStyledLinearAds` **[boolean][35]** See usage also in [Handling VPAID Ads][39] (optional, default `false`)
+* `adsRenderingSettings.useStyledLinearAds` **[boolean][35]** See usage also in [Handling VPAID Ads][39] (optional, default `false`)
 * `companions` **[CompanionsConfigObject][40]?** Defines the companion ads.
 * `locale` **[string][40]?** Sets the publisher provided locale.
+* `adsRequest.vastLoadTimeout` **[number][32]** Override for default VAST load timeout in milliseconds for a single wrapper. The default timeout is 5000ms.
```

Comment:
1. should be `vastLoadTimeout` (remove `adsRequest`)
2. run docs:generate again (what is [32]? it should be a link to this property)